src/main/resources/templates/order_list.html

```diff
@@ -19,60 +19,30 @@ <h1 class="mt-5 mb-3 text-center">Zamówienia</h1>
         <table class="table table-striped table-bordered">
             <thead class="table-dark">
             <tr>
-                <th></th>
```

Comment:
Now the user sees all orders, not just his own. Mbe instead of edit existing endpoint, make another named for example `user_orders`. One will be for admin panel and one for user then.